### PR TITLE
Add String List

### PR DESCRIPTION
**Pull request by SergiyBt**

### CODE REVIEW

File: List.h [hunk 1]
```diff
@@ -0,0 +1,255 @@
+#pragma once
+#include <stdlib.h>
+#include <string.h>
+
+void StringListInit(char*** list) {
```
Comment (lysak-v) [inline List.h]: The open curly brace should be at new line

**Comment by AIaska** [comment]
The code is written in accordance with code convention.

File: List.h [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+	
+	list = nullptr;
+	list = (char***)malloc(2 * sizeof(char**));
```
Comment (yuriikostiuk) [inline List.h]: should be checked allocating memory

File: List.h [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+	const size_t SIZE = StringListSize(list);
+
+	for (size_t i = 0; i != SIZE; ++i)
+	{
+		if (strcmp(*list[i], str) == 0)
+			return i;
+	}
+
+	return SIZE;
```
Comment (lysak-v) [inline List.h]: You should check the case when your list haven't such string

Comment (SergiyBt) [reply List.h]: I check it in main function if return value equals size of my list it means that list hasn"t such string

Comment (SergiyBt) [reply List.h]: I check it in main function if return value equals size of my list it means that list hasn"t such string

**Pull request closed without merging**